### PR TITLE
HPCC-18028 Improve the aesthetics of the login page

### PR DESCRIPTION
**Pull request by kunalaswani**
Improve the aesthetics to the login page by changing background colors and text fields.

Signed-off-by: Kunal Aswani <kunal.aswani@lexisnexis.com>

## Type of change:
- [ ] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [x] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [x] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [x] The commit message title makes sense in a changelog, by itself.
  - [x] The commit is signed.
- [x] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [x] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [x] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [x] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [x] I have given due consideration to all of the following potential concerns:
  - [x] Scalability
  - [x] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [x] There are no similar instances of the same problem that should be addressed
  - [x] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [x] This is a user interface / front-end modification
  - [x] I have tested my changes in multiple modern browsers
  - [x] The component(s) render as expected

### CODE REVIEW

**Comment by miguelvazq** [comment]
Please complete checklist.

File: esp/src/eclwatch/templates/Login.html [hunk 1]
```diff
@@ -19,7 +20,8 @@
                 width: 500px;
                 padding: 20px 0 20px 0;
                 border-radius: 25px;
-                border: 2px solid #eaeaea;
+                border: 1px solid #bfbfbf;
+                background-color: #eaeaea;
```
Comment (miguelvazq) [inline esp/src/eclwatch/templates/Login.html]: This should be #fff;

**Comment by kunalaswani** [comment]
@miguelvazq please review, changes made and squashed the commit.

File: esp/src/eclwatch/templates/Login.html [hunk 2]
```diff
@@ -19,7 +20,8 @@
                 width: 500px;
                 padding: 20px 0 20px 0;
                 border-radius: 25px;
-                border: 2px solid #eaeaea;
+                border: 1px solid #fff;
```
Comment (miguelvazq) [inline esp/src/eclwatch/templates/Login.html]: .container did not need to be modified .formContainer did.

Comment (miguelvazq) [inline esp/src/eclwatch/templates/Login.html]: `.formContainer {
                width: 500px;
                padding: 20px 0 20px 0;
                border-radius: 25px;
                border: 1px solid #bfbfbf;
                background-color: #fff;
                margin: auto;
            }`

**Review by miguelvazq — COMMENTED** [review]
Added comments

**Comment by miguelvazq** [comment]
Looks good.

**Comment by kunalaswani** [comment]
@richardkchapman ready to be merged

**Comment by richardkchapman** [comment]
@kunalaswani Please squash

**Pull request merged**